### PR TITLE
[DPEDE-2163] Update change button to edit in expansion panel

### PR DESCRIPTION
**Pull request by emerson-pereira**
https://ctl.atlassian.net/browse/DPEDE-2163

### CODE REVIEW

**Review by wojciechwlodarczyklumen — APPROVED** [review]
Looks okay !

**Review by wojciechwlodarczyklumen — APPROVED** [review]

File: test/chi/chi-vue/expansion-panel.pug [hunk 1]
```diff
@@ -26,8 +26,8 @@ h2 Expansion Panel - Tests
       chi-button(color='primary' @click="active += 1" data-chi-epanel-action='next')
         | Continue
     template(v-slot:change)
-      chi-button(color='primary' variant='flat' @click="active = 0" data-chi-epanel-action='change')
-        | Change
+      chi-button(color='primary' variant='flat' @click="active = 0" data-chi-epanel-action='edit')
```
Comment (josecarlosgonzalezv) [inline test/chi/chi-vue/expansion-panel.pug:29]:
```suggestion
      chi-button(color='primary' variant='flat' @click="active = 0" data-chi-epanel-action='change')
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]
Please review that you only change **names** not panel actions, slots or whatever

File: test/chi/chi-vue/expansion-panel.pug (hunk 1 above)
Comment (emerson-pereira) [reply test/chi/chi-vue/expansion-panel.pug]: Done

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**